Assignment3/CarlosJM7-y160190/CarlosJM7-CarlosJimenez.txt

```diff
@@ -0,0 +1,177 @@
+
+SELECT DISTINCT ?property
+WHERE 
+{
+	?x a <http://dbpedia.org/ontology/Politician> .
+	{?isValueOf ?property ?x}
+	UNION
+	{?x ?property ?hasValue}
+	FILTER(!(?property = rdf:type))
+}
+
+RESULT:
+ "property" ,
+ "http://www.w3.org/2002/07/owl#differentFrom" ,
+ "http://www.w3.org/2000/01/rdf-schema#seeAlso" ,
+ "http://www.w3.org/2002/07/owl#sameAs" ,
+ "http://dbpedia.org/ontology/wikiPageWikiLink" ,
+ "http://dbpedia.org/ontology/wikiPageRedirects" ,
+ "http://dbpedia.org/ontology/wikiPageDisambiguates" ,
+ "http://dbpedia.org/property/name" ,
+ "http://xmlns.com/foaf/0.1/primaryTopic" ,
+ "http://dbpedia.org/ontology/academicAdvisor" ,
+ "http://dbpedia.org/ontology/appointer" ,
+ "http://dbpedia.org/ontology/associate" ,
+ "http://dbpedia.org/ontology/author" ,
+ "http://dbpedia.org/ontology/award" ,
+ "http://dbpedia.org/ontology/basedOn" ,
+ "http://dbpedia.org/ontology/builder" ,
+ "http://dbpedia.org/ontology/chancellor" ,
+ "http://dbpedia.org/ontology/child" ,
+ "http://dbpedia.org/ontology/commander" ,
+ "http://dbpedia.org/ontology/creator" ,
+ "http://dbpedia.org/ontology/deputy" ,
+
+
+
+3. Which different values exist for the properties, except rdf:type, of the instances of the Politician class?
+
+SELECT DISTINCT ?property ?hasValue
+WHERE 
+{
+	?x a <http://dbpedia.org/ontology/Politician> .
+	?x ?property ?hasValue .
+	FILTER(!(?property = rdf:type))
+}
+
+RESULT:
+ "property" , "hasValue" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Axel Oxenstierna" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , أكسل أكسنسترنا ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Axel Oxenstierna" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Axel Oxenstierna" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Axel Oxenstierna" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Axel Oxenstierna" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , アクセル・オクセンシェルナ ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Axel Oxenstierna" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Axel Oxenstierna" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Axel Oxenstierna" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Оксеншерна, Аксель" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Brothers Grimm" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , الأخوان غريم ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Brüder Grimm" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Hermanos Grimm" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Frères Grimm" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Fratelli Grimm" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , グリム兄弟 ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Gebroeders Grimm" ,
+ "http://www.w3.org/2000/01/rdf-schema#label" , "Bracia Grimm" ,
+
+
+4. For each of the properties, except rdf:type, that can be applied to instances of the Politician class, which different values do they take in those instances?
+
+SELECT DISTINCT ?property ?isValue
```

Comment:
We were not actually referring to the instances that are related through a property to instances of politician.